lib/src/date_page_view.dart

```diff
@@ -52,19 +51,56 @@ class _DatePageViewState extends State<DatePageView> {
           axisDirection: AxisDirection.right,
           offset: position,
           anchor: 0,
-          slivers: <Widget>[
-            SliverFillViewport(
-              viewportFraction: 1 / visibleDays,
-              delegate: SliverChildBuilderDelegate(
-                (context, index) => widget.builder(
-                  context,
-                  LocalDate.fromEpochDay(index + visibleDays ~/ 2),
-                ),
-              ),
-            ),
-          ],
+          center: _centerKey,
+          slivers: _slivers,
         );
       },
     );
   }
+
+  List<Widget> get _slivers {
```

Comment:
Due to the complexity of this block, I prefer a method instead of a simple getter
```suggestion
  List<Widget> _createSlivers() {
```

Reply:
maybe `_buildSlivers()` ?

Reply:
changed to `_buildSlivers()` method